target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -599,6 +600,7 @@ ath79_setup_macs()
 	dlink,dap-3662-a1)
 		label_mac=$(mtd_get_mac_ascii bdcfg "wlanmac")
 		;;
+	trendnet,tew-673gru|\
```

Comment:
Please maintain alphabetical order.

Reply:
Thanks for catching that (and apologies you needed to).